src/popper/runner_host.py

```diff
@@ -485,7 +485,13 @@ def _build_from_recipe(self, build_ctx_path, build_dest, cid):
         pwd = os.getcwd()
         os.chdir(build_ctx_path)
         recipefile = SingularityRunner._get_recipe_file(build_ctx_path, cid)
-        self._s.build(recipe=recipefile, image=cid, build_folder=build_dest, force=True)
+        self._s.build(
+            recipe=recipefile,
+            image=cid,
+            build_folder=build_dest,
+            force=True,
+            quiet=self._s.quiet,
```

Comment:
cool! it's nice that there's already a `quiet` parameter on the singularity python lib side. The only thing to note here is that the value for this parameter should be the one from `self._config.quiet`, which is the one that holds the value provided from the CLI